collective/jsonify/wrapper.py

```diff
@@ -744,6 +744,21 @@ def get_zope_dublin_core(self):
         else:
             self['modification_date'] = ''
 
+    def get_basic_dates(self):
+        """
+        """
+        if self._is_cmf_only_obj():
+            # then the dates are handled by get_zope_dublin_core,
+            # so we do nothing.
+            return
+        # datetime fields
+        for field in ['creation_date', 'modification_date']:
```

Comment:
effective and expired are already covered, since they are in the plone.publication behavior, and therefore are included from iterSchemata line 111

Reply:
Should this also include effective date?

Reply:
@ale-rt see my comment above : effective and expired are already covered, since they are in the plone.publication behavior, and therefore are included from iterSchemata line 111 (and same for archetypes)